modules/oracle-xe/src/main/java/org/testcontainers/containers/OracleContainer.java

```diff
@@ -90,6 +90,11 @@ public OracleContainer withPassword(String password) {
         return self();
     }
 
+    @Override
+    public OracleContainer withUrlParam(String paramName, String paramValue) {
```

Comment:
Some databases evidently don't support passing URL parameters. But DBs like Oracle definitely support passing parameters through a `Properies` object when creating the connection. While that is out of scope for this commit, I would however suggest that we already know consider if we want to use the same `AbstractJdbcContainer` method for both purposes or not. In that case we might want to drop the `Url` part and rename the method to `withParam()`, `withJdbcParam()`, or similar, so that client code doesn't have to call `withUrlParam()` for some DBs and then possibly `withProperiesParam()` for others. Just something worth considering.